tests/vnc_test.go

```diff
@@ -53,22 +53,28 @@ import (
 	"kubevirt.io/client-go/log"
 	"kubevirt.io/client-go/subresources"
 
+	"kubevirt.io/kubevirt/pkg/util"
 	launcherApi "kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 
 	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/framework/checks"
 	"kubevirt.io/kubevirt/tests/framework/kubevirt"
 	"kubevirt.io/kubevirt/tests/libvmifact"
 	"kubevirt.io/kubevirt/tests/libwait"
 )
 
-var _ = Describe("[rfe_id:127][crit:medium][arm64][vendor:cnv-qe@redhat.com][level:component][sig-compute]VNC", decorators.SigCompute, func() {
+var _ = Describe("[rfe_id:127][crit:medium][arm64][vendor:cnv-qe@redhat.com][level:component][sig-compute]VNC", decorators.SigCompute, decorators.WgS390x, func() {
 
 	var vmi *v1.VirtualMachineInstance
 
 	Describe("[rfe_id:127][crit:medium][vendor:cnv-qe@redhat.com][level:component]A new VirtualMachineInstance", func() {
 		BeforeEach(func() {
 			var err error
 			vmi = libvmifact.NewGuestless()
+			arch := util.TranslateBuildArch()
+			if checks.IsS390X(arch) {
+				vmi = libvmifact.NewAlpine()
```

Comment:
Why do these specific tests need to be Alpine?

IOW what is this VMI providing on s390x that the guestless VMI can't?

The answer to these questions should be documented as code comments here IMHO.